ExternalLookup/src/main/java/cheetah/example/externallookup/function/ExternalLookupMapper.java

```diff
@@ -51,6 +56,15 @@ public void asyncInvoke(InputEvent externalLookupInputEvent, ResultFuture<Output
     public void open(Configuration parameters) {
         client = HttpClient.newHttpClient();
 
+        tokenProvider = KeyedTokenProvider.getInstance();
+        tokenProvider.registerTokenProviderIfAbsent(TOKEN_ID,
```

Comment:
this formatting looks a bit odd, but im not sure if it actually gets any prettier. maybe with end parentheses on separate lines?